python_modules/jupedsim/jupedsim/distributions_app.py

```diff
@@ -139,14 +139,14 @@ def main():
 
         for i in range(circle_count):
             if style == "density":
-                densities.append(st.slider(f"Agents / m² in Circle segment {i+1}", 0.1,
-                                           round(((1371.2855*0.0648**distance_to_agents) + 100)/100, 2), key=i), )
+                densities.append(st.slider(f"Agents / m² in Circle segment {i + 1}", 0.1,
```

Comment:
Please add a comment about the equation for max denisty. How did you calculate it?

Reply:
function was calculated by Power-Regression
done